migrations/versions/40ee030e9302_v23_6_feature_1377.py

```diff
@@ -0,0 +1,627 @@
+"""v23_6_feature_1377
+
+Revision ID: 40ee030e9302
+Revises: ca8ff8c0c01d
+Create Date: 2023-04-07 11:07:01.686463
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+
+from app.database import get_db_schema, engine
+from app.model.db import IDXPosition, ExecutableContract, IDXPositionBondBlockNumber
+
+# revision identifiers, used by Alembic.
+revision = "40ee030e9302"
+down_revision = "ca8ff8c0c01d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    connection = op.get_bind()
+
+    ############################
+    # Migration for position
+    ############################
+    op.get_bind().execute(sa.text(f"DELETE FROM position WHERE token_address IS null;"))
+    op.get_bind().execute(
+        sa.text(f"DELETE FROM position WHERE account_address IS null;")
+    )
+    op.get_bind().execute(sa.text(f"DELETE FROM position WHERE modified IS null;"))
+    op.get_bind().execute(
+        sa.text(
+            f"""DELETE FROM position WHERE (modified) NOT IN (
+                    SELECT latest_position.max_modified AS modified FROM (
+                        SELECT  max(modified) AS max_modified FROM position GROUP BY token_address, account_address
+                    ) AS latest_position
+                );"""
+        )
+    )
```

Comment:
- To remove duplicate rows, added SQL statement to erase all but the most recent data. 
- Whether the data is up-to-date or not is determined by **modified** field.
- Such SQL is executed against the following tables.
  - `position`
  - `executable_contract`
  - `idx_position_bond_block_number`
  - `idx_position_share_block_number `
  - `idx_position_coupon_block_number `
  - `idx_position_membership_block_number `